### ISSUE TITLE
Canvas redraw in iOS 11

### ISSUE THREAD

**Issue by regrex** [apache/incubator-echarts#7011]
### Canvas redraw in iOS 11 [问题简述]
We use canvas to draw a radar map, but, in iOS 11, When first show in the webview, the page looks fine, but, when it scrolls down and the canvas in it becomes invisible in the viewport, the content in the canvas disappears for a moment, then reappear. 
![img_0061](https://user-images.githubusercontent.com/10026094/32646763-1392e56e-c629-11e7-9b17-73ef1087bf62.png)
![img_0060](https://user-images.githubusercontent.com/10026094/32646764-1515f3a4-c629-11e7-9b0f-2bf74a82efa8.png)

### Version & Environment [版本及环境]
+ ECharts version [ECharts 版本]: 3.8.0
+ Browser version [浏览器类型和版本]: Webview
+ OS Version [操作系统类型和版本]: iOS 11

### Expected behaviour [期望结果]
not disappear

### ECharts option [ECharts配置项]

```javascript
option = {
    backgroundColor: '#fff',
    radar:{
        indicator:indicator,
        splitNumber:1,
        silent: true,
        nameGap:5,
        name: {
            textStyle: {
                color: '#333',
                fontSize: '12px'
            }
        },
        splitLine: {
            lineStyle: {
                color: ['#999']
            }
        },
        axisLine: {
            lineStyle: {
                type: 'dotted',
                color: ['#999']
            }
        }
    },
    series:[{
        type: 'radar',
        symbol: 'none',

        itemStyle: {
            normal: {
                color: '#F5A623'
            }
        },
        areaStyle: {
            normal: {
                color: 'rgba(255,225,0,0.60)'
            }
        },
        animationDelay:1000,
        animationDuration: 3000,
        data: [{
            value: valuePercent
        }]
    }]
}
```

**Comment by Ovilia** [comment]
How can you know the content disappears when canvas is invisible?

**Comment by cijianzy** [comment]
Same problem, when you scroll up after you scroll down, It will from disappear become appear or not appear at all.

**Comment by zl19921218** [comment]
相同的问题  在ios 11 上当画布滑动到不可见区域后  图表会消失   除了监听图标是否在页面可见区域内重新渲染图标  还有什么更好的办法吗   ？？？？？？？？？

**Comment by CYvonne** [comment]
我也遇到同样的问题了，救命啊(╥╯^╰╥)

**Comment by liuzhibin123** [comment]
我也遇到了 楼主那样的问题，待解决。

**Comment by mlxiao93** [comment]
最后怎么解决的

**Comment by regrex** [comment]
用WKWebView，而不是UIWebView，可以解决iOS11上这类问题，包括fixed元素滚动的时候消失的问题。或者等到iOS 11.3（用beta版试过，怀疑iOS11.3的webview用了WKWebView，或者做了升级）

**Comment by ouhuiju** [comment]
meet the same problem. I use the WKWebView in my project, when we load the url included canvas and scroll, the canvas will disappear sometime, but check the DOM it always on!..
